apps/vscode/extension/src/unfurl.ts

```diff
@@ -1,4 +1,4 @@
-import cheerio from 'cheerio'
+import { load } from 'cheerio'
```

Comment:
ooc, does `load` prevent some of the the transitive packages to load? (less package size)?

Reply:
i doubt it